Source/JavaScriptCore/Scripts/process-entitlements.sh

```diff
@@ -112,23 +112,47 @@ elif [[ "${WK_PLATFORM_NAME}" == macosx ]]
 then
     [[ "${RC_XBS}" != YES ]] && plistbuddy Add :com.apple.security.get-task-allow bool YES
 
-    if [[ "${PRODUCT_NAME}" == jsc ]]; then mac_process_jsc_entitlements
+    if [[ "${PRODUCT_NAME}" == jsc ||
+          "${PRODUCT_NAME}" == dynbench ||
+          "${PRODUCT_NAME}" == minidom ||
+          "${PRODUCT_NAME}" == testair ||
+          "${PRODUCT_NAME}" == testb3 ||
+          "${PRODUCT_NAME}" == testdfg ||
+          "${PRODUCT_NAME}" == testmasm ||
+          "${PRODUCT_NAME}" == testmem ||
+          "${PRODUCT_NAME}" == testRegExp ]]; then mac_process_jsc_entitlements
     elif [[ "${PRODUCT_NAME}" == testapi ]]; then mac_process_testapi_entitlements
```

Comment:
I'm a little curious why the other test targets do not use the `testapi` entitlements path. AFAICT, the only difference is that testapi gets a `com.apple.rootless.storage.JavaScriptCore` entitlement and jsc does not.

Reply:
Yes, `com.apple.rootless.storage.JavaScriptCore` is the only difference.  I was erring on the side of since these tests didn't need it, I chose to go with the `jsc` entitlements which leave it out.